tests/test_username_api.py

```diff
@@ -1,9 +1,9 @@
 import json
 import logging
 import os.path
-import random
 import string
 
+import exrex
```

Comment:
Move this below the `from ... import ...` just for styling reasons

Reply:
alright